schemas/target/target-classification.schema.json

```diff
@@ -0,0 +1,98 @@
+{
+  "meta:license": [
+    "Copyright 2023 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/schemas/target-classification",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe Target Classification Fields",
+  "description": "A set of meta-data fields that will classify Target activities and experiences.",
+  "type": "object",
+  "meta:extensible": false,
+  "meta:abstract": false,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "definitions": {
+    "target": {
+      "properties": {
+        "xdm:activityName": {
+          "title": "Activity Name",
+          "type": "string",
+          "description": "The name of the Target activity."
+        },
+        "xdm:experienceName": {
+          "title": "Experience Name",
+          "type": "string",
+          "description": "The name of the Target experience."
+        },
+        "xdm:activityExperience": {
+          "title": "Activity Experience",
+          "type": "string",
+          "description": "Target Activity name concatenated with experience name."
+        },
+        "xdm:thirdId": {
+          "title": "Third Id",
+          "type": "string",
+          "description": "The third id of the classification record."
+        },
+        "xdm:activityId": {
+          "title": "Activity Id",
+          "type": "string",
+          "description": "The identity of Target activity."
+        },
+        "xdm:experienceId": {
+          "title": "Experience Id",
+          "type": "string",
+          "description": "The identity of Target experience."
+        },
+        "xdm:analyticsReporting": {
+          "title": "Analytics Reporting",
+          "type": "string",
+          "description": "The reporting type associated with the Target activity."
+        },
+        "xdm:activeDates": {
```

Comment:
If its not an array, then why plurals ? Please rename this to activityDate

Also, please add a format="date-time"

Reply:
The string is a combination of start date and end date, separated by '|', hence activedates plural.

Reply:
We could create two new fields to separate the two dates. It would make them more usable.

Reply:
agreed with @shagreel . Having semantically well defined  dates is much better than concatenated strings which make them less usable downstream as dates.

Reply:
If the decision as per slack discussion is to keep it like this then I would recommend to rename this to activityStartAndEndDate